Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -59,6 +59,26 @@ struct SymbolGraphLoader {
         let bundle = self.bundle
         let dataProvider = self.dataProvider
         
+        /// Computes the default availbiality based on the `inheritDefaultAvailability` option.
+        let defaultAvailabilities: ([DefaultAvailability.ModuleAvailability]?) -> [DefaultAvailability.ModuleAvailability]? = { defautAvailabilities in
+            guard let defautAvailabilities else { return nil }
+            // Check the selected behaviour for inheritance of the default availability and remove the avaialbity
+            // version if it's set to  `platformOnly`.
+            if
+                let applyDefaultAvailabilityVersionToSymbols = bundle.info.inheritDefaultAvailability,
+                applyDefaultAvailabilityVersionToSymbols == .platformOnly {
```

Comment:
nit: This is weirdly formatted. It looks strange with the condition on a different line and if statements with multiple lines of conditions are more readable if they move the `{` to its own line to separate the conditions from the body.

Generally I would format this like 
```swift
if let applyDefaultAvailabilityVersionToSymbols = bundle.info.inheritDefaultAvailability,
   applyDefaultAvailabilityVersionToSymbols == .platformOnly 
{
    ...
```

but here it looks like `applyDefaultAvailabilityVersionToSymbols` is only used in the second condition, so they could be combined:
```suggestion
            if bundle.info.inheritDefaultAvailability == .platformOnly {
```

Reply:
Resolved in https://github.com/swiftlang/swift-docc/pull/1024/commits/6055aa94f29b2e827aafa8598f531adbeccb25e7